src/Sulu/Bundle/MediaBundle/Media/FormatCache/FormatCacheClearer.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\FormatCache;
+
+/**
+ * Format cache clearer.
+ */
+class FormatCacheClearer
```

Comment:
yes every format cache should be clearable. sometimes you change the compression of a format then you need to clear the format cache.